IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory/ngsa-memory.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
```

Comment:
this confused me at first - should we move the second name: ngsa above labels:?  I thought it was a typo when I first looked at it

Reply:
moved the second name: ngsa up for clarity